src/rdb.c

```diff
@@ -3141,6 +3126,24 @@ int rdbLoadRioWithLoadingCtx(rio *rdb, int rdbflags, rdbSaveInfo *rsi, rdbLoadin
                 if (isbase) serverLog(LL_NOTICE, "RDB is base AOF");
             } else if (!strcasecmp(auxkey->ptr, "redis-bits")) {
                 /* Just ignored. */
+            } else if (!strcasecmp(auxkey->ptr, "slot-info")) {
+                int slot_id;
+                unsigned long slot_size, expires_slot_size;
+                /* Try to parse the slot information. in case the number of parsed arguments is smaller than expected
+                 * Will fail the RDB load */
```

Comment:
Nit
```suggestion
                /* Try to parse the slot information. In case the number of parsed arguments is smaller than expected
                 * we'll fail the RDB load. */
```